data_validation/__main__.py

```diff
@@ -491,10 +488,34 @@ def store_yaml_config_file(args, config_managers):
     cli_tools.store_validation(config_file_path, yaml_configs)
 
 
-def run(args):
-    """ """
-    config_managers = build_config_managers_from_args(args)
+def partition_and_store_config_files(args: Namespace) -> None:
+    """Build multiple YAML Config files using user specified partition logic
+
+    Args:
+        args (Namespace): User specified Arguments
 
+    Returns:
+        None
+    """
+    # Default Validate Type
+    validate_cmd = consts.ROW_VALIDATION
```

Comment:
Instead of declaring this separately, we can pass in the constant directly to the next line.

Reply:
Done